src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$column-width: 100px;
+$gap: 1px;
+
+.calendar {
+  display: flex;
+  width: calc((7 * $column-width) + (6 * 1px));
+  flex-wrap: wrap;
+  gap: $gap;
+  box-sizing: border-box;
+
+  &__day {
+    box-sizing: border-box;
+    border: solid black 1px;
+    background-color: #eee;
+    width: $column-width;
+    height: 100px;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: transform 300ms;
+
+    &:hover {
+      transform: translate(0, -20px);
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transition: transform 300ms;
```

Comment:
You have defined a `transition` property on the `&__day` element at line 30 already. There's no need to redefine it within the `&:hover` state unless you are changing the property values. If the transition property is the same, you can remove it from here to avoid redundancy.